lib/renaissance/auctions/auction.ex

```diff
@@ -21,24 +23,15 @@ defmodule Renaissance.Auction do
     auction
     |> cast(attrs, @required_fields ++ @optional_fields)
     |> validate_required(@required_fields)
-    |> validate_price()
-    |> validate_date()
+    |> foreign_key_constraint(:seller_id, name: :auctions_user_id_fkey)
+    |> Constraint.amount_constraint(:price)
+    |> end_datetime_constraint()
```

Comment:
`_constraint` implies something being applied in the database that Ecto should know about. This should be `validate_end_datetime`

Reply:
+1